pkgs/development/python-modules/ilcli/default.nix

```
@@ -0,0 +1,34 @@
+{
+  buildPythonPackage,
+  fetchFromGitHub,
+  lib,
+  setuptools,
+  wheel,
+}:
+
+buildPythonPackage rec {
+  pname = "ilcli";
+  version = "unstable-2020-07-22";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "cloudant";
+    repo = "ilcli";
+    rev = "2c033240a18603dd99c2dd8f6185ad0f0169c8c7";
```

```suggestion
    # no tags
    rev = "2c033240a18603dd99c2dd8f6185ad0f0169c8c7";
```
Please leave a comment explaining why we need the commit hash.